first_ruby/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Whats this file?

Reply:
Used by Yarn which is a package manager to specify what npm dependencies needed for Rails application.

Reply:
Aside from yarn, what else can use this?

Reply:
NPM will be using this besides yarn.

Reply:
nonesense.

npm & yarn is the same kind of tool.
you can use npm install instead of yarn install and all will be working as normal.